pkg/karmadactl/get/get.go

```diff
@@ -578,20 +571,25 @@ func (g *CommandGetOptions) reconstructObj(obj runtime.Object, mapping *meta.RES
 	}
 
 	for rowIdx := range table.Rows {
-		var tempRow metav1.TableRow
-		rbKey := getRBKey(mapping.GroupVersionKind, table.Rows[rowIdx], cluster)
-
+		var cells []interface{}
 		if g.OutputWatchEvents {
-			tempRow.Cells = append(append(tempRow.Cells, event, table.Rows[rowIdx].Cells[0], cluster), table.Rows[rowIdx].Cells[1:]...)
+			cells = append(append(cells, event, table.Rows[rowIdx].Cells[0], cluster), table.Rows[rowIdx].Cells[1:]...)
 		} else {
-			tempRow.Cells = append(append(tempRow.Cells, table.Rows[rowIdx].Cells[0], cluster), table.Rows[rowIdx].Cells[1:]...)
+			cells = append(append(cells, table.Rows[rowIdx].Cells[0], cluster), table.Rows[rowIdx].Cells[1:]...)
 		}
-		if _, ok := RBInfo[rbKey]; ok {
-			tempRow.Cells = append(tempRow.Cells, "Y")
-		} else {
-			tempRow.Cells = append(tempRow.Cells, "N")
+		table.Rows[rowIdx].Cells = cells
+
+		unObj := &unstructured.Unstructured{}
+		err := unObj.UnmarshalJSON(table.Rows[rowIdx].Object.Raw)
+		if err != nil {
+			klog.Errorf("Failed to unmarshal unObj, error is: %v", err)
+			continue
+		}
+		v, exist := unObj.GetLabels()[karmadautil.ManagedByKarmadaLabel]
+		if exist && v == karmadautil.ManagedByKarmadaLabelValue {
+			table.Rows[rowIdx].Cells = append(table.Rows[rowIdx].Cells, "Y")
 		}
```

Comment:
Here need a `continue`?  or `Y` and `N` can appear at the same time?

Reply:
Thanks for reminding.